packages/react/carbon.yml

```diff
@@ -0,0 +1,699 @@
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-datatable
+    date-picker:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/date-picker/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-datepicker
+    dropdown:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/dropdown/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-dropdown
+    error-boundary:
+      name: Error boundary
+      framework: react
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-errorboundary
+      # TODO: @Aubrey - double check tags
+      tags:
+        - system-feedback
+    file-uploader:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/file-uploader/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-fileuploader--file-uploader
+    flex-grid:
+      name: Flex grid
+      framework: react
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-flexgrid
+      # TODO: @Aubrey - double check tags
+      tags:
+        - shell
+    form:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/form/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-form
+    form-group:
+        name: Form group
+        framework: react
+        status: stable
+        type: component
+        platform: web
+        # TODO: add thumbnail
+        thumbnailPath: '' 
+        demoLinks:
+          - type: storybook
+            name: Storybook
+            action: link
+            url: https://react.carbondesignsystem.com/?path=/docs/components-formgroup
+        # TODO: @Aubrey - double check tags
+        tags:
+          - form
+          - input-control  
```

Comment:
Delete input-control. A form is a group of related input controls that allow users to provide data or configure options. We would put input-control on something that is simplified, like a standalone button or text input field.